.github/workflows/i18n-weekly-pull.yaml

```diff
@@ -1,5 +1,10 @@
 name: i18n-weekly-pull
 
+env:
+  # See: https://github.com/actions/setup-go/tree/main#supported-version-syntax
+  GO_VERSION: ">=1.18"
```

Comment:
Why 1.18 and not 1.19?
Should we specify something like `^1.19.4` to get the latest 1.19?

Reply:
Good catch on the `1.18` 👍 
For the specific version, I first went that way but then I saw that the `setup-go` action [already picks the latest minor release](https://github.com/arduino/arduino-cli/actions/runs/3846152753/jobs/6551134726#step:4:15). So I steered towards keeping the same pattern as before